packages/tools/devtools/devtools-core/src/data-visualization/DefaultEditors.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This module contains default {@link EditSharedObject}
+ * implementations for our DDSs.
+ */
+
+import { SharedCell } from "@fluidframework/cell";
+import { SharedCounter } from "@fluidframework/counter";
+import { SharedString } from "@fluidframework/sequence";
+
+import { ISharedObject } from "@fluidframework/shared-object-base";
+import { Serializable } from "@fluidframework/datastore-definitions";
+import { EditSharedObject } from "./DataEditing";
+
+/**
+ * Default {@link EditSharedObject} for {@link SharedCell}.
+ */
+export const editSharedCell: EditSharedObject = async (
+	sharedObject: ISharedObject,
+	newData: Serializable,
+): Promise<void> => {
+	const sharedCell = sharedObject as SharedCell<unknown>;
+	const data = sharedCell.get();
+	console.log(data);
```

Comment:
Is there anything preventing us from actually applying the edit to the cell here? Seems like we should be able to call `sharedCell.set` without issue (unless I'm missing something).

Reply:
I think @t-santiagode simply consoled it (at the moment) since we `SharedCell` requires more thorough type checking (unlike `SharedString` `SharedCounter`).

Reply:
I was thinking that could be part of the next PR. I would like to couple the drop-down and and SharedCell together, and I would like to atleast have PR out. I could include it in this but, that would be the focus for the next one. Thoughts?

Reply:
We'll also want to accompany any editors we add with unit tests 🙂

Reply:
We talked about this being pushed off to a later PR

Reply:
Yup. Feel free to resolve, just be sure to file a work item to track it :)